### ISSUE TITLE
feature.ToShape() returns null

### ISSUE THREAD

**Issue by pedwik** [DotSpatial/DotSpatial#917]
*IF BUG, INCLUDE THIS PART:*

### Steps to reproduce

Converting a feature with ToShape() raises an exception, because constructor Shape(Ifeature f) includes the assignment _attributes = feature.DataRow.ItemArray. It may happen that feature.DataRow is intentionally not set when creating features programmatically. Also, the comment for the constructor Shape(feauture) says "Creates a shape based on the specified feature. This shape will be standing alone, all by itself. The fieldnames and field types will be null.". So I think checking feature.DataRow for null before invoking it is appropiate.
```
var coordinates = new List<Coordinate>() 
{
     new Coordinate(1375930, 6269230),
     new Coordinate(1376248, 6269230),
     new Coordinate(1376248, 6268860),
     new Coordinate(1375930, 6268860),
};
var feature = new Feature(FeatureType.Polygon, coordinates);
var shape = feature.ToShape();
```
DotSpatial version: 1.9

**Comment by mogikanin** [comment]
Agree, inside Shape ctor when assigning `_attributes` should be check that `feature.DataRow` is not null. @pedwik would you like provide pull request with this change (with unit-test)?
